src/ToDoForm.jsx

```diff
@@ -1,53 +1,115 @@
 import React, { useState } from 'react';
+import { Button, TextField, Select, MenuItem, styled } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import theme from './Theme';
+
+import ToDo from './ToDo';
 
 function ToDoForm({ onAddTodo }) {
     const [userInput, setUserInput] = useState({
         task: '',
         date: '1',
+        priority: ToDo.priorities.LOW,
     });
 
     const handleChange = (e) => {
-        setUserInput({ task: e.currentTarget.value, date: userInput.date });
+        setUserInput({ ...userInput, task: e.target.value });
     };
     const handleChangeDate = (e) => {
-        setUserInput({ date: e.currentTarget.value, task: userInput.task });
+        setUserInput({ ...userInput, date: e.target.value });
+    };
+    const handleChangePriority = (e) => {
+        setUserInput({ ...userInput, priority: e.target.value });
     };
 
     const handleSubmit = (e) => {
         e.preventDefault();
 
         onAddTodo(userInput);
-        setUserInput({ task: '', date: '1' });
+        setUserInput({ task: '', date: '1', priority: ToDo.priorities.LOW });
     };
+    function priorityColor(props) {
+        if (props === ToDo.priorities.LOW) return theme.palette.priority.low;
+        if (props === ToDo.priorities.MEDIUM)
+            return theme.palette.priority.medium;
+
+        return theme.palette.priority.high;
+    }
+    const PriorityMenuItem = styled(MenuItem)((props) => ({
+        color: priorityColor(props.color),
+    }));
+    const styles = { backgroundColor: theme.palette.form.field };
     return (
         <>
             <h3> New Task:</h3>
+
             <form onSubmit={handleSubmit}>
-                <input
+                <TextField
+                    label="enter task..."
+                    style={styles}
```

Comment:
only use `style` is a last resort for customization.

when to use customization?
- when component styling API does not support the styling i want

how to customize?
- sx prop (if the styling is unique and not reusable)
- styled component (if styling is reusable)
- overrides (irrelevant for now but you can read about it)

https://mui.com/material-ui/customization/how-to-customize/#1-one-off-customization